Simplenote/Classes/SPNoteEditorViewController.m

```diff
@@ -1982,8 +2032,66 @@ - (void)viewVersionAction:(id)sender {
                                                      delegate:self];
     versionActionSheet.tapToDismiss = NO;
     versionActionSheet.cancelButtonIndex = 0;
+    
+    [self disableRotation];
```

Comment:
It looks like it was disabled there because the Versions UI doesn't update properly. The layout is mostly fine but the selection gets a bit funky.

If you meant for the Info UI, it works there. I **never** use my phone in landscape so it didn't even occur to me to **not** disable it! :) — I've enabled rotation for the Info UI now.